util/network/client.py

```diff
@@ -13,7 +13,7 @@ class ClientUtils:
 
     @staticmethod
     def start(url):
-        sio.connect(url, wait=False)
+        sio.connect(url)

```

Comment:
we're starting the client in a thread now, so waiting or not doesn't really matter anymore